include/vapor/SetHDF5PluginPath.h

```diff
@@ -3,20 +3,16 @@
 #include "vapor/MyBase.h"
 #include "vapor/ResourcePath.h"
 
-#ifndef WIN32
-    #include <H5PLpublic.h>
-#endif
-
 namespace VAPoR {
 void SetHDF5PluginPath() {
-    string plugins = Wasp::GetSharePath("plugins");
-
-    #ifndef WIN32
-        H5PLreplace(plugins.c_str(), 0);
+    int rc;
```

Comment:
Good software engineering practice: not to leave a variable not initialized.
Suggested change: int rc = 0;

Reply:
Gotcha.